pkg/infra/pulumi_aws/iac/k8s/add_ons/fluent_bit_for_eks/index.ts

```diff
@@ -0,0 +1,80 @@
+import * as pulumi_k8s from '@pulumi/kubernetes'
+import { Eks } from '../../../eks'
+
+export const installFluentBitForCW = (eks: Eks, provider) => {
+    const namespaceName = 'amazon-cloudwatch'
+    const label = { name: namespaceName }
+    const cwNamespace = eks.createNamespace(namespaceName, label)
+
+    const configMap = eks.getClusterName().apply((name) => {
+        return new pulumi_k8s.core.v1.ConfigMap(
+            `${eks.clusterName}-fluent-bit-cluster-info`,
+            {
+                metadata: {
+                    name: 'fluent-bit-cluster-info',
+                    namespace: namespaceName,
+                },
+                data: {
+                    'cluster.name': name,
+                    'logs.region': eks.region,
+                    'http.server': 'On',
+                    'http.port': '2020',
+                    'read.head': 'Off',
+                    'read.tail': 'On',
+                },
+            },
+            { provider, dependsOn: [cwNamespace] }
+        )
+    })
+
+    new pulumi_k8s.yaml.ConfigFile(
+        `${eks.clusterName}-FluentBitDriver`,
+        {
+            file: 'https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/latest/k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluent-bit/fluent-bit.yaml',
```

Comment:
Is it normal to be reading the config file directly from a samples repo?

Reply:
yeah its fairly normal. When we do a lot of the k8s work well likely pull these back in, but its going to be very messy to do this in pulumi. Anyways, not an abnormal patter for people to follow, thats essentially what youre doing with helm anyways